cypress/integration/popover/position.feature

```diff
@@ -1,26 +1,28 @@
 Feature: Popover positioning
 
     Background:
-        Given a popover with a width of 360px and height of 200px
-        And an anchor element that renders a popover when a user clicks the anchor
+        Given the popover has a width of 360px and height of 200px
+
+    Scenario: Spacing between anchor and popover
+        When the anchor is clicked
+        Then there is some space between the anchor and the popover
 
     Scenario: Default positioning
-        Given the anchor has more than 200px above, before the limit of the <body> element
-        When the popover is rendered
+        Given the space above the anchor is more than the limit
```

Comment:
Can we define 'the limit', perhaps in the background? It seems ambiguous to me.

Reply:
See my explanation above. We can add comments, but I wouldn't add it to the spec definitions.

Reply:
ping

Reply:
This one I def. would like to keep unclear as that's not part of the behavior.

Reply:
Done, see https://github.com/dhis2/ui-core/pull/527#issuecomment-562426939